tests/testAutopas/tests/autopasInterface/AutoPasInterfaceTest.cpp

```diff
@@ -346,7 +346,9 @@ void testSimulationLoop(const autopas::Configuration &conf) {
   // do first simulation loop
   doSimulationLoop(autoPas, &functor);
   doAssertions(autoPas, &functor, numParticles, __LINE__);
-  moveParticlesAndResetF({autoPas.getVerletSkin() / 6, 0., 0.});
+
+  // forcing the particle to move inwards
+  moveParticlesAndResetF({-autoPas.getVerletSkin() / 6, 0., 0.});
```

Comment:
Note to whoever picks up this PR: This was done because there is bad handling if a particle gets moved outside the container and a rebuild occurs (which is the case here due to the buffer particle threshold stuff). This does not appear to be directly related to this PR but pre-existing bad behaviour.